### ISSUE TITLE
Problem with trainning in levi_graph mode

### ISSUE THREAD

**Issue by bjwfc** [emorynlp/levi-graph-amr-parser#5]
Hi han,
  i followed your steps to train the model using levi graph for several times.But they all lead to the warning, which the specific content is "arc_loss = *** exploded! please retrain **". Could you give me some advice to advoid this situation or i just need retrain for several times?
  Thank you !

**Comment by hankcs** [comment]
Hi,

Yes, you just need to retrain for another time whenever you see this warning. Sometimes the loss exceeds a large threshold and the resulted model performs significantly worse on the dev set. I don't have time to debug into the codes as lots of thirdparty codes are involved. So don't waste time to continue training if you see the warning.

**Comment by bjwfc** [comment]
Hi han，
thanks for your prompt response!I will follow your advice to retrain the model.
Hope you have a nice day!
